### DISCUSSION TITLE
Left/Right Audio Pan Adjustment

### DISCUSSION THREAD

**Discussion by Coasternaut** [f1multiviewer/issue-tracker#30] [Feature requests] [answered]
Could there be an option to adjust the left/right audio balance on each player? I like to listen to the international feed on one side and the F1 Live on the other and this would make it a lot easier.

**Comment by f1multiviewer** [comment, marked as answer]
Not really I think, but you can use a virtual audio device and then configure the audio output for each stream to go to a different output (right click) and wire it up so it's left/right channel. On macOS you could use https://rogueamoeba.com/loopback/, I'm sure Windows has an alternative too.

**Reply by Coasternaut** [reply]
Okay, I figured out a way using some different virtual cable programs and OBS because there aren't any good free audio mixers for Mac. I use Voicemeeter on Windows and it's pretty good.
